pkg/daemon/update.go

```diff
@@ -122,6 +122,28 @@ func (dn *Daemon) performPostConfigChangeAction(postConfigChangeActions []string
 	return dn.triggerUpdateWithMachineConfig(state.currentConfig, state.desiredConfig, true)
 }
 
+func setRunningKargsWithCmdline(config *mcfgv1.MachineConfig, requestedKargs []string, cmdline []byte) error {
+	splits := splitKernelArguments(strings.TrimSpace(string(cmdline)))
+	config.Spec.KernelArguments = nil
```

Comment:
this shouldn't be necessary?

Reply:
Wanted to make sure that calling twice wouldn't cause it to behave badly.